.changeset/olive-cars-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search': patch
+---
+
+Create an experimental search plugin that is compatible with declarative integration system.
```

Comment:
I think it can be good to mention here that it's exported via `/alpha`

Reply:
Good idea, done here: https://github.com/backstage/backstage/pull/20169/commits/b62aaa3a68abf6ce0810961c3195b3c1e1ad5a42